### ISSUE TITLE
Specific css for a container fails installation

### ISSUE THREAD

**Issue by poul-erik** [ChrisHammond/DNNTemplates#66]
I have created a container with a specific css. I have include the css in the dnn file:

![billede](https://user-images.githubusercontent.com/38832672/124902503-47329680-dfe3-11eb-8394-05b8ef9f5a6f.png)

But when I attempt to install, I get an error. Seams the installer cannot find the css in the package (apparently it should be in the root?)
![billede](https://user-images.githubusercontent.com/38832672/124910117-82d15e80-dfeb-11eb-8508-7578797937a8.png)

I modified the SkinPackage.targets build script:

![billede](https://user-images.githubusercontent.com/38832672/124902173-fde24700-dfe2-11eb-95a8-049d193e1764.png)

I don't know if that is a correct solution, but it fixed the installation problem.

Regards

Poul Erik Sørensen

**Comment by poul-erik** [comment]
I also had problems with thumbnails for the containers. To solve this I had to also add the corresponding jpg file as a container file to the dnn file.

![billede](https://user-images.githubusercontent.com/38832672/124951933-1880e400-e014-11eb-8310-6cc0af11e251.png)

I'm pretty sure that this is not the correct solution. When I look at the skin part. The packaging manage to include both the ascx, the corresponding css file as well as the corresponding jpg file. 

When it comes to the container I have to manually handle these and I also noticed that the Container.css file is not included. For the skin package the Skin.css is included without any manual intervention.

Regards

Poul Erik Sørensen
